.github/workflows/functions-v2-helloauditlog.html

```diff
@@ -0,0 +1,66 @@
+name: functions-v2-helloauditlog
```

Comment:
files extension should be updated to YAML

Reply:
Done.